### ISSUE TITLE
Giara fails to start: RuntimeError: could not create new GType: giara+giara_clamp+GiaraClamp (subclass of AdwClamp)

### ISSUE THREAD

**Issue by klemensn** [NixOS/nixpkgs#168842]
### Describe the bug
giara fails to start but merely prints a python backtrace.

### Steps To Reproduce
1. $ `nix-shell -p giara` or add "pkgs.giara" to `environment.systemPackages`
   and rebuild the system
2. $ giara

```console
/nix/store/dixynjprqk0fzbhkryv6j049svkx6b98-python3.9-pygobject-3.42.0/lib/python3.9/site-packages/gi/types.py:217: Warning: cannot derive 'giara+giara_clamp+GiaraClamp' from final parent type 'AdwClamp'
  _gi.type_register(cls, namespace.get('__gtype_name__'))
Traceback (most recent call last):
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/bin/..giara-wrapped-wrapped", line 68, in <module>
    from giara import __main__
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/__main__.py", line 8, in <module>
    from giara.app_window import AppWindow
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/app_window.py", line 4, in <module>
    from giara.main_ui import MainUI
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/main_ui.py", line 4, in <module>
    from giara.main_deck import MainDeck
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/main_deck.py", line 2, in <module>
    from giara.left_stack import LeftStack
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/left_stack.py", line 3, in <module>
    from giara.sections import PostListSection
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/sections.py", line 2, in <module>
    from giara.post_preview_listview import PostPreviewListView
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/post_preview_listview.py", line 5, in <module>
    from giara.giara_clamp import GiaraClampScrollable
  File "/nix/store/aqji0xafsdyzsd7yc7jiy33ylckbcc12-giara-1.0/lib/python3.9/site-packages/giara/giara_clamp.py", line 4, in <module>
    class GiaraClamp(Adw.Clamp):
  File "/nix/store/dixynjprqk0fzbhkryv6j049svkx6b98-python3.9-pygobject-3.42.0/lib/python3.9/site-packages/gi/types.py", line 226, in __init__
    super(GObjectMeta, cls).__init__(name, bases, dict_)
  File "/nix/store/dixynjprqk0fzbhkryv6j049svkx6b98-python3.9-pygobject-3.42.0/lib/python3.9/site-packages/gi/types.py", line 205, in __init__
    cls._type_register(cls.__dict__)
  File "/nix/store/dixynjprqk0fzbhkryv6j049svkx6b98-python3.9-pygobject-3.42.0/lib/python3.9/site-packages/gi/types.py", line 217, in _type_register
    _gi.type_register(cls, namespace.get('__gtype_name__'))
RuntimeError: could not create new GType: giara+giara_clamp+GiaraClamp (subclass of AdwClamp)
```

### Expected behavior
Start up the graphical reddit client.

### Notify maintainers
@dasj19

### Metadata
```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.32, NixOS, 22.05 (Quokka), 22.05pre368234.fd364d26885`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.7.0`
 - channels(root): `"nixos"`
 - channels(kn): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by dasj19** [comment]
Thanks for reporting and sorry for the answering delay.

I have looked into it and there is no easy fix for version 1.0 and version 1.0.1 requires a new dependency which is not yet in nixpkgs. I'll take a second look at it when time allows.

**Comment by jtojnar** [comment]
Yeah, the package requires an older version of libadwaita. And 1.0.1 requires https://gitlab.gnome.org/jwestman/blueprint-compiler.
